client/go/outline/platerrors/error_code.go

```diff
@@ -24,6 +24,9 @@ package platerrors
 // You can reliably use these values in TypeScript to check for specific errors.
 type ErrorCode string
 
+// An [ErrorCode] that will only be used internally.
+const unknownError ErrorCode = "[ERR_UNKNOWN]"
```

Comment:
Please remove the `[]` from the error code. We can add them for printing if wanted.

```suggestion
const unknownError ErrorCode = "ERR_UNKNOWN"
```

Reply:
I removed this entire `ErrorCode`.